serverless/networking/serverless-ossm-tls.adoc

```diff
@@ -0,0 +1,165 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-tls"]
+= Configuring TLS for a custom domain using {ProductName}
+:context: serverless-ossm-tls
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+.Prerequisites
+* Install xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[{ServerlessProductName}].
+* Install xref:../../service_mesh/service_mesh_install/preparing-ossm-installation.adoc#preparing-ossm-installation[{ProductName}].
+* In this example, `openssl` is used to generate certificates, but you can use any certificate generation tool you like to create these.
+
+.Procedure
+
+. Create TLS certificates:
++
+
+[source,terminal]
+----
+$ openssl req -x509 -sha256 -nodes -days 365 -newkey rsa:2048 -subj '/O=Example Inc./CN=example.com' -keyout example.com.key -out example.com.crt
+----
+
++
+
+[source,terminal]
+----
+$ openssl req -out custom.example.com.csr -newkey rsa:2048 -nodes -keyout custom.example.com.key -subj "/CN=custom-ksvc-domain.example.com/O=Example Inc."
+----
+
++
+
+[source,terminal]
+----
+$ openssl x509 -req -days 365 -CA example.com.crt -CAkey example.com.key -set_serial 0 -in custom.example.com.csr -out custom.example.com.crt
+----
+
+. Check that the certificates appear in your directory:
++
+
+[source,terminal]
+----
+$ ls -1
+----
+
++
+
+.Example output
+[source,terminal]
+----
+custom.example.com.crt
+custom.example.com.csr
+custom.example.com.key
+example.com.crt
+example.com.key
+----
+
+. Create a secret:
++
+
+[source,terminal]
+----
+$  oc create -n istio-system secret tls custom.example.com --key=custom.example.com.key --cert=custom.example.com.crt
+----
+
+. Attach the secret to the Istio ingress gateway, by editing the `ServiceMeshControlPlane` resource.
+.. Edit the `ServiceMeshControlPlane` resource:
++
+
+[source,terminal]
+----
+$ oc edit -n istio-system ServiceMeshControlPlane <control_plane_name>
+----
+
+.. Check that the following lines exist in the resource, and if they do not, add them:
++
+
+[source,yaml]
+----
+istio-ingressgateway:
+  secretVolumes:
+  - mountPath: /custom.example.com
+    name: custom-example-com
+    secretName: custom.example.com
+----
+
+. Update the Istio ingress gateway to use your secret.
+.. Edit the `default-gateway` resource:
++
+
+[source,terminal]
+----
+$ oc edit gateway default-gateway
+----
+
+.. Check that the following lines exist in the resource, and if they do not, add them:
```

Comment:
I tried to follow additional docs. They were added to git in previous PR?! They are not clear on what exactly to do. I was trying to use `demo` namespace instead of `default` and docs doesn't say what is required to be changed. I wasn't able to pass with verification below this point.

Reply:
Do you mean the custom domain docs? If so, yes this was added in a previous PR because these TLS docs weren't requested until after the other service mesh docs.

@nak3 can you please work with Chris to verify the docs and lmk if we need changes to anything that was previously added in the _Networking_ section to make these docs work? Thanks.